numpy/ma/tests/test_core.py

```diff
@@ -1990,6 +2001,205 @@ def test_datafriendly_mul_arrays(self):
         assert_equal(a, [[1, 1], [3, 3]])
         assert_equal(a.mask, [[0, 1], [0, 1]])
 
+    def test_inplace_addition_scalar_type(self):
+        # Test of inplace additions
+        for t in self.othertypes:
```

Comment:
This would like generate a `DeprecationWarning` or `TypeError` as the types will mismatch. This is exactly what we want to avoid.